### ISSUE TITLE
Is token padding or attention_mask needed and supported for RWKV?

### ISSUE THREAD

**Issue by zjersey** [BlinkDL/RWKV-LM#154]
I read the code of RWKV and found there is no padding ids for input.

How should RWKV model work for those padding tokens when training?

**Comment by BlinkDL** [comment]
it's not using padding, but you can modify dataset.py to do that
